### ISSUE TITLE
Queue Tab doesn't work

### ISSUE THREAD

**Issue by Lukeluha** [rabbitmq/rabbitmq-management#530]
I use RabbitMQ 3.72 with management plugin in Docker. When I go to the tab queue, I only get error "ReferenceError: fmt_op_policy_short is not defined ReferenceError: fmt_op_policy_short is not defined at Array.process".

Previous versions runs correctly.

RabbitMQ version: 3.7.2
Erlang version: 20.1.7

**Comment by michaelklishin** [comment]
Thank you for your time.

Team RabbitMQ uses GitHub issues for specific actionable items engineers can work on. This assumes two things:

1. GitHub issues are not used for questions, investigations, root cause analysis, discussions of potential issues, etc (as defined by this team)
2. We have a certain amount of information to work with

We get at least a dozen of questions through various venues every single day, often quite light on details.
At that rate GitHub issues can very quickly turn into a something impossible to navigate and make sense of even for our team. Because of that questions, investigations, root cause analysis, discussions of potential features are all considered to be mailing list material by our team. Please post this to [rabbitmq-users](https://groups.google.com/forum/#!forum/rabbitmq-users).

Getting all the details necessary to reproduce an issue, make a conclusion or even form a hypothesis about what's happening can take a fair amount of time. Our team is multiple orders of magnitude smaller than the RabbitMQ community. Please help others help you by providing a way to reproduce the behavior you're
observing, or at least sharing as much relevant information as possible [on the list](https://groups.google.com/forum/#!forum/rabbitmq-users):

 * Server, client library and plugin (if applicable) versions used
 * Server logs
 * A code example or terminal transcript that can be used to reproduce
 * **Full** exception stack traces (not a single line message)
 * `rabbitmqctl status` (and, if possible, `rabbitmqctl environment` output)
 * Other relevant things about the environment and workload, e.g. [a traffic capture](https://www.rabbitmq.com/amqp-wireshark.html)

Feel free to edit out hostnames and other potentially sensitive information.

When/if we have enough details and evidence we'd be happy to file a new issue.

Thank you.

**Comment by michaelklishin** [comment]
I cannot reproduce and this hasn't been reported elsewhere to my knowledge. Consider clearing your browser cache.

**Comment by Tensho** [comment]
Usually, people consider on "cache cleaning" only cookies cleanup, but there is also required "Local Storage" and "Session Storage" cleanup. Maybe it's worth to note that details in the documentation either. I ran into the same issue described here from the first attempt.

**Comment by michaelklishin** [comment]
@Tensho you are right. The UI does use browser storage for a few things. Please submit a PR that extends the browser cache part of the Upgrade guide?

**Comment by Tensho** [comment]
Sure, rabbitmq/rabbitmq-website#507

**Comment by fake-name** [comment]
I'm seeing this as well, after a bunch of messy updates (see https://github.com/rabbitmq/rabbitmq-server/issues/1668). Clearing the local storage *does not* fix it.

**Comment by fake-name** [comment]
This appears to be a client-side issue:

```
[Fri Aug 10 2018 22:34:34 GMT-0700 (Pacific Daylight Time)] body 404 Not Found get /#/queues  Error: 404 Not Found get /#/queues 
    at Sammy.Application.error (sammy-0.7.6.min.js:5)
    at Sammy.Application.notFound (sammy-0.7.6.min.js:5)
    at Sammy.Application.runRoute (sammy-0.7.6.min.js:5)
    at Sammy.Application._checkLocation (sammy-0.7.6.min.js:5)
    at Sammy.Application.run (sammy-0.7.6.min.js:5)
    at start_app_login (main.js:67)
    at HTMLDocument.<anonymous> (main.js:3)
    at i (jquery-1.12.4.min.js:2)
    at Object.fireWith [as resolveWith] (jquery-1.12.4.min.js:2)
    at Function.ready (jquery-1.12.4.min.js:2)
main.js:1029 Uncaught error: ReferenceError: fmt_op_policy_short is not defined
main.js:1030 Stack: ReferenceError: fmt_op_policy_short is not defined
    at Array.process (eval at compile (https://rabbit.wlnupdates.com/js/ejs-1.0.min.js:1:6654), <anonymous>:114:52)
    at EJS.render (https://rabbit.wlnupdates.com/js/ejs-1.0.min.js:1:1885)
    at format (https://rabbit.wlnupdates.com/js/main.js:1026:21)
    at https://rabbit.wlnupdates.com/js/main.js:412:24
    at with_reqs (https://rabbit.wlnupdates.com/js/main.js:1008:9)
    at https://rabbit.wlnupdates.com/js/main.js:1004:17
    at XMLHttpRequest.req.onreadystatechange (https://rabbit.wlnupdates.com/js/main.js:1084:17)
```

**Comment by fake-name** [comment]
Wait, never mind. I cleared the browser cache using the chrome dev tools, and it didn't fix it. 

In the process of debugging the issue via the network tab of the chrome dev tools, I hit "disable cache", and the issue was resolved. 

So https://github.com/rabbitmq/rabbitmq-website/pull/507 isn't *quite* right. The problem isn't local storage, it's the browser having some of the js files cached, and the various jerberscript dependencies having version discordance.
